proto/streamlit/proto/NewSession.proto

```diff
@@ -121,6 +124,33 @@ message CustomThemeConfig {
   string text_color = 4;
   FontFamily font = 5;
   BaseTheme base = 6;
+  string widget_background_color = 7;
+  string widget_border_color = 8;
+  Radii radii = 9;
+  string body_font = 13;
+  string code_font = 14;
+  repeated FontFace font_faces = 15;
+  FontSizes font_sizes = 16;
+}
+
+message FontFace {
+  string url = 1;
+  string family = 2;
+  int32 weight = 3;
+  string style = 4;
+}
+
+message Radii {
+  // In pixels.
+  int32 baseWidgetRadius = 1;
+  int32 checkboxRadius = 2;
+}
+
+message FontSizes {
+  // In pixels.
+  int32 tinyFontSize = 1;
+  int32 smallFontSize = 2;
+  int32 baseFontSize = 3;
```

Comment:
We'll need to give @sfc-gh-wschmitt a heads up that a bunch of fields are being added here, but I'm assuming this is okay because we probably just nuke the theme `custom_theme` field entirely when sanitizing forward messages